### PR TITLE
Load module by id should also get the style

### PR DESCRIPTION
**Pull request by yozmag**
loadmoduleid plugin keyword changed to get the module chrome style.

Pull Request for Issue # .

### Summary of Changes

Changed to allow the chrome style with loadmoduleid keyword

### Expected result
The chrome style should change if the plugin parameter changes or a 2nd argument is defined. 

### Actual result

Nothing changes always the none chrome is render.

### CODE REVIEW

File: plugins/content/loadmodule/loadmodule.php [hunk 1]
```diff
@@ -129,8 +129,15 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
 		{
 			foreach ($matchesmodid as $match)
 			{
-				$id     = trim($match[1]);
-				$output = $this->_loadid($id);
+				$matchesidlist = explode(',', $match[1]);
+				$id     = trim($matchesidlist[0]);
+				// We may not have a module style so fall back to the plugin default.
```
Comment (ReLater) [inline plugins/content/loadmodule/loadmodule.php:134]: Add an empty line before comment

File: plugins/content/loadmodule/loadmodule.php [hunk 2]
```diff
@@ -217,13 +224,13 @@ protected function _loadmod($module, $title, $style = 'none')
 	 *
 	 * @since   3.9.0
 	 */
-	protected function _loadid($id)
+	protected function _loadid($id $style = 'none')
```
Comment (ReLater) [inline plugins/content/loadmodule/loadmodule.php]: Missing comma
```suggestion
	protected function _loadid($id, $style = 'none')
```

File: plugins/content/loadmodule/loadmodule.php [hunk 3]
```diff
@@ -217,13 +224,13 @@ protected function _loadmod($module, $title, $style = 'none')
...
 	{
 		self::$modules[$id] = '';
 		$document = JFactory::getDocument();
 		$renderer = $document->loadRenderer('module');
 		$modules  = JModuleHelper::getModuleById($id);
-		$params   = array('style' => 'none');
+		$params = array('style' => $style);
```
Comment (ReLater) [inline plugins/content/loadmodule/loadmodule.php]: Indent
```suggestion
		$params   = array('style' => $style);
```

File: plugins/content/loadmodule/loadmodule.php [hunk 4]
```diff
@@ -129,8 +129,15 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
 		{
 			foreach ($matchesmodid as $match)
 			{
-				$id     = trim($match[1]);
-				$output = $this->_loadid($id);
+				$matchesidlist = explode(',', $match[1]);
+				$id     = trim($matchesidlist[0]);
```
Comment (ReLater) [inline plugins/content/loadmodule/loadmodule.php]: Indent
```suggestion
				$id            = trim($matchesidlist[0]);
```

File: plugins/content/loadmodule/loadmodule.php [hunk 5]
```diff
@@ -129,8 +129,15 @@ public function onContentPrepare($context, &$article, &$params, $page = 0)
...
+				// We may not have a module style so fall back to the plugin default.
+				if (!array_key_exists(1, $matchesidlist))
+				{
+					$matchesidlist[1] = $style;
+				}
```
Comment (ReLater) [inline plugins/content/loadmodule/loadmodule.php:138]: Add empty line after
```suggestion
				}

```

**Comment by brianteeman** [comment]
Won't this impact existing sites?

**Comment by ReLater** [comment]
And isn't it a new feature that won't go into Joomla 3? I'm not sure because pr is for Joomla 3.10 branch.

**Comment by yozmag** [comment]
If a site now uses loadmoduleid it loads the 'none' chrome'. If the parameter on the plugin is different option '-1' https://docs.joomla.org/Understanding_Output_Overrides#Module_Chrome the plugin is going to render the chrome select in the plugin parameter instead of 'none' and will impact the render of the module.
This Pr should go to 3.10 version or next 3.9... Thanks.

**Comment by ReLater** [comment]
> This Pr should go to 3.10 version or next 3.9... Thanks.

I think your pr is a new feature and not a bug fix. New features won't go into Joomla 3. You have to rebase it to Joomla 4.

But let's decide the 3.9 Release Lead @HLeithner

**Comment by HLeithner** [comment]
It's seams not to be a b/c break but a new feature, so sry please rebase of J4.

**Comment by yozmag** [comment]
I think this is a bug because the parameter is on the plugin is not taken to account when render the module like with keyword  {loadposition xxxx} and  {loadmodule xxxx} the styles (modChrome_style) is working. Also adding the style as parameter also works as described in joomla documentation . The load {loadmoduleid xx} was added feature on the 3.9 this doesn't make sense not to make correct only on J4, and J3. stays with the bug.

**Pull request closed without merging**